### PR TITLE
Fix version code for android APKs

### PR DESCRIPTION
**Pull request by jherico**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16152/HighFidelity-Beta-PR14640-843a5d9.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16152/HighFidelity-Beta-PR14640-843a5d9.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16152/HighFidelity-Beta-Interface-PR14640-843a5d9.dmg

File: android/containerized_build.sh [hunk 1]
```diff
@@ -9,14 +9,19 @@ docker run \
...
+   -e ANDROID_BUILD_TARGET \
+   -e ANDROID_BUILD_DIR \
+   -e CMAKE_BACKTRACE_URL \
+   -e CMAKE_BACKTRACE_TOKEN \
+   -e CMAKE_BACKTRACE_SYMBOLS_TOKEN \
+   -e GA_TRACKING_ID \
+   -e OAUTH_CLIENT_SECRET \
+   -e OAUTH_CLIENT_ID \
+   -e OAUTH_REDIRECT_URI \
+   -e VERSION_CODE \
```
Comment (sethalves) [inline android/containerized_build.sh]: GIT_PR_COMMIT dropped on purpose?

File: android/containerized_build.sh [hunk 2]
```diff
@@ -9,14 +9,19 @@ docker run \
    --rm \
     --security-opt seccomp:unconfined \
    -v "${WORKSPACE}":/home/jenkins/hifi \
-   -e "RELEASE_NUMBER=${RELEASE_NUMBER}" \
-   -e "RELEASE_TYPE=${RELEASE_TYPE}" \
-   -e "ANDROID_BUILD_TARGET=assembleDebug" \
```
Comment (sethalves) [inline android/containerized_build.sh]: are we sure that ANDROID_BUILD_TARGET is in  the environment?

**Pull request merged**